# Halt only on errors, because of
# doc/ja/user-guide/http_smart_server.txt:161: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
diff -urNad bzr-2.5.1-orig/Makefile bzr-2.5.1/Makefile
--- bzr-2.5.1-orig/Makefile	2012-05-22 18:10:04.000000000 +0300
+++ bzr-2.5.1/Makefile	2013-03-03 20:06:46.000000000 +0200
@@ -220,7 +220,7 @@
 # support our "plain" html documentation so that Sphinx is not a hard
 # dependency for packagers on older platforms.
 
-rst2html = $(PYTHON) tools/rst2html.py --link-stylesheet --footnote-references=superscript --halt=warning
+rst2html = $(PYTHON) tools/rst2html.py --link-stylesheet --footnote-references=superscript --halt=error
 
 # translate txt docs to html
 derived_txt_files = \
